src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/EkspanderbartBegrunnelsePanel.tsx

```diff
@@ -33,6 +34,7 @@ const StyledExpansionTitle = styled(ExpansionCard.Title)`
 
 interface IEkspanderbartBegrunnelsePanelProps extends PropsWithChildren {
     vedtaksperiodeMedBegrunnelser: IVedtaksperiodeMedBegrunnelser;
+    sisteFom?: string;
```

Comment:
Kunne vi ha funnet et litt mer beskrivende navn? Måtte lete i koden for å skjønne hva det betydde 😆

Reply:
Absolutt! Hva tenker du at er mer beskrivende?